styleguide/gulpfile.js

```diff
@@ -208,13 +207,38 @@ gulp.task("svg2twig", function() {
 });
 
 // Copy twig files from source
-/* copy files */
 gulp.task("copyTwigFiles", function() {
   return gulp.src(config.twigSource.files)
     .pipe(plumber())
     .pipe(gulp.dest(config.twigSource.dest))
 });
 
+gulp.task('cleanTwig', ['clean:before'], function (callback) {
```

Comment:
In the rest of the file we're using hyphens to denote multi-word commands. Lets use that instead of the camel case to keep it consistent.